deploy/manager/dev/50_manager_deployment.yaml

```diff
@@ -36,6 +36,13 @@ spec:
         volumeMounts:
         - mountPath: /mnt/etc/scylla-manager
           name: scylla-manager-config
+        readinessProbe:
+          httpGet:
+            path: /api/v1/clusters
+            port: 5080
+          initialDelaySeconds: 60
```

Comment:
does it really start that long when the scyllacluster is already up?

Reply:
removed delay